client/my-sites/site-settings/form-seo.jsx

```diff
@@ -532,31 +600,42 @@ export const SeoForm = React.createClass( {
 							{ hasError( 'yandex' ) && this.getVerificationError( showPasteError ) }
 						</FormFieldset>
 						<FormFieldset>
-							<FormLabel htmlFor="seo_sitemap">{ this.translate( 'XML Sitemap' ) }</FormLabel>
-							<ExternalLink className="seo-sitemap" icon={ true } href={ sitemapUrl } target="_blank">{ sitemapUrl }</ExternalLink>
+							<FormLabel htmlFor="seo_sitemap">{ translate( 'XML Sitemap' ) }</FormLabel>
+							<ExternalLink
+								className="seo-sitemap"
+								icon={ true }
+								href={ sitemapUrl }
+								target="_blank">
+								{ sitemapUrl }
+							</ExternalLink>
 							<FormSettingExplanation>
-								{ this.translate( 'Your site\'s sitemap is automatically sent to all major search engines for indexing.' ) }
+								{ translate( 'Your site\'s sitemap is automatically sent to all major search engines for indexing.' ) }
 							</FormSettingExplanation>
 						</FormFieldset>
 					</Card>
 				</form>
 				<WebPreview
 					showPreview={ showPreview }
-				    onClose={ this.hidePreview }
-				    previewUrl={ siteUrl }
-				    showDeviceSwitcher={ false }
-				    showExternal={ false }
-				    defaultViewportDevice="seo"
+					onClose={ this.hidePreview }
+					previewUrl={ siteUrl }
+					showDeviceSwitcher={ false }
+					showExternal={ false }
+					defaultViewportDevice="seo"
 				/>
 			</div>
 		);
 	}
 } );
 
-const mapStateToProps = state => ( {
-	selectedSite: getSelectedSite( state ),
-	storedTitleFormats: getSeoTitleFormatsForSite( getSelectedSite( state ) )
-} );
+const mapStateToProps = ( state, ownProps ) => {
+	const { site = getSelectedSite( state ) } = ownProps;
+
+	return {
+		site,
+		storedTitleFormats: getSeoTitleFormatsForSite( site ),
+		showAdvancedSeo: site && site.plan && hasBusinessPlan( site.plan ) && isEnabled( 'manage/advanced-seo' )
+	};
+};
 
 const mapDispatchToProps = {
 	refreshSiteData: siteId => requestSite( siteId ),
```

Comment:
There's also a query component for this:
```javascript
{ siteId && <QuerySites siteId={ siteId } /> }
```

Just be careful to not pass through a falsey value as siteId, or we'll fetch all sites.